app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
```

Comment:
- [ ] You have done a great job. As the [previous reviewer commented](https://github.com/andym80/Blog-app-Rails/pull/4/files#r1035338463) I will suggest you create the `show.html.erb` view for this show action.